newIDE/app/src/InstancesEditor/CompactInstancePropertiesEditor/index.js

```diff
@@ -149,6 +149,7 @@ const CompactInstancePropertiesEditor = ({
     >
       <ScrollView
         autoHideScrollbar
+        style={{ paddingTop: marginsSize }}
         key={instances
```

Comment:
Alternative could be to wrap the content of the scroll view in a Line?

Reply:
I tried but it creates this width issue and the editor width does not follow the mosaic panel one